src/main.cpp

```diff
@@ -517,13 +536,15 @@ void __assert_fail(const char* assertion, const char* file, unsigned int line, c
 int main(int argc, char** argv)
 {
     int timeout = 10;
+    bool enable_debug_printing = false;
 
     Core::ArgsParser args_parser;
     args_parser.set_general_help("LibJS test262 runner for streaming tests");
     args_parser.add_option(s_harness_file_directory, "Directory containing the harness files", "harness-location", 'l', "harness-files");
     args_parser.add_option(s_use_bytecode, "Use the bytecode interpreter", "use-bytecode", 'b');
     args_parser.add_option(s_parse_only, "Only parse the files", "parse-only", 'p');
     args_parser.add_option(timeout, "Seconds before test should timeout", "timeout", 't', "seconds");
+    args_parser.add_option(enable_debug_printing, "Enable debug printing", "debug", 'd');
```

Comment:
Hold on, this is in the runner directly... how does the `main.py` script interface with this option, i.e. how do you use it?

Reply:
I use it when running the runner directly, I can add an option to the main.py script but that probably gets quite spammy. Should I add an option there as well?

Reply:
Done